docs/guides/working-with-the-file-system.md

```diff
@@ -258,6 +258,47 @@ await webcontainerInstance.mount(files, { mountPoint: 'my-mount-point' });
 
 :::
 
+## Generating snapshots
+
+The `mount()` method not only accepts the tree-like format we described above, but a binary snapshot format that can be automatically generated, using the
+[`@webcontainer/snapshot`](https://www.npmjs.com/package/@webcontainer/snapshot) package.
+Suppose we want our WebContainer API application to mount a source code folder that is present on our server. In that case, we could do the following:
+
+```typescript
+import { snapshot } from '@webcontainer/snapshot';
+
+// snapshot is a `Buffer`
+const snapshot = await snapshot(SOURCE_CODE_FOLDER);
```

Comment:
```suggestion
const folderSnapshot = await snapshot(SOURCE_CODE_FOLDER);
```

(making this suggestion, because the code is invalid otherwise as `snapshot` is used both as a const variable and a named import)